build-tools.Dockerfile

```diff
@@ -65,6 +69,10 @@ COPY --from=pgcopydb_builder /pgcopydb/lib/libpq.so.5 /pgcopydb/lib/libpq.so.5
 #
 # 'gdb' is included so that we get backtraces of core dumps produced in
 # regression tests
+RUN echo 'Acquire::Retries "5";' > /etc/apt/apt.conf.d/80-retries && \
+    echo -e "retry_connrefused = on\ntimeout=15\ntries=5\n" > /root/.wgetrc \
+    echo -e "--retry-connrefused\--conntect-timeout 15\nn--retry 5\n--max-time 300\n" > /root/.curlrc
```

Comment:
Should we have all these lines for all images even if some of them don't use curl/wget at this point?

Do we want to add the same retries for `nonroot` user?

Reply:
as for the `nonroot`: thanks for spotting that, I will add it there.

Reply:
as for having these lines for all images: well, we just yesterday had discussion with @jcgruenhage about "dead code", and so far we aligned with the approach:
- nice to have one place as example we can copy code from when needed,
- do it when needed.
- reasoning: imagine we set now some defaults for `curl` and `wget`. Next iteration someone could go adding default for other tools. And so it easy to get code clogged with something that not going to be used
- and also could add extra questions for someone who is new with the code, kind of: hey, I see here settings for `wget`, `aws` and `helm`, but can't see where it runs, did I miss something?

for sure both approaches have pros and cons. should I add these lines for all images, please let me know.

Reply:
> please let me know.

I'll leave that up for you to decide :)